backend/internal/oauth/oauth2/token/tokenhandler.go

```diff
@@ -225,6 +231,13 @@ func (th *tokenHandler) HandleTokenRequest(w http.ResponseWriter, r *http.Reques
 		IDToken:      tokenRespDTO.IDToken.Token,
 	}
 
+	// For token exchange, set the issued_token_type from context
+	if grantType == constants.GrantTypeTokenExchange {
+		if val, ok := ctx.TokenAttributes["issued_token_type"].(string); ok {
+			tokenResponse.IssuedTokenType = val
+		}
```

Comment:
Do we need default value if the type assertion fails? 
Spec mandates having issued_token_type in the response?

Reply:
Is putting this to ctx is the correct approach ?
Can we use the tokenRespDTO ?

Reply:
We have returned a model.TokenResponse object
```
tokenResponse := &model.TokenResponse{
		AccessToken:  tokenRespDTO.AccessToken.Token,
		TokenType:    tokenRespDTO.AccessToken.TokenType,
		ExpiresIn:    tokenRespDTO.AccessToken.ExpiresIn,
		RefreshToken: tokenRespDTO.RefreshToken.Token,
		Scope:        scopes,
		IDToken:      tokenRespDTO.IDToken.Token,
	}
```

I would suggest if this is the only place we use `issued_token_type`, then use a object to get it without setting it to ctx